web/portal/platform/cypress/pacts/platform-provider-login-platform-consumer-login.json

```diff
@@ -1 +1,85 @@
-{"consumer":{"name":"platform-consumer-login"},"provider":{"name":"platform-provider-login"},"interactions":[{"description":"logout and login fails with wrong password","providerState":"","request":{"method":"POST","path":"/api/platform/admin_login","headers":{"accept":"application/json, text/plain, */*","content-length":"38","content-type":"application/x-www-form-urlencoded"},"body":"username=admin&password=wrong-password","query":""},"response":{"status":401,"headers":{"content-type":"application/json"},"body":{"code":401,"message":"Invalid credentials."},"matchingRules":{"$.body.code":{"match":"type"},"$.body.message":{"match":"type"}}}},{"description":"login","providerState":"","request":{"method":"POST","path":"/api/platform/admin_login","headers":{"accept":"application/json, text/plain, */*","content-length":"38","content-type":"application/x-www-form-urlencoded"},"body":"username=irontec&password=ee4aeVie7AiH","query":""},"response":{"status":200,"headers":{"content-type":"application/json"},"body":{"token":"token_token_token","refresh_token":"refresh_token"},"matchingRules":{"$.body.token":{"match":"type"},"$.body.refresh_token":{"match":"type"}}}}],"metadata":{"pactSpecification":{"version":"2.0.0"},"client":{"name":"pact-cypress-adapter","version":"1.3.0"}}}
\ No newline at end of file
+{
+  "consumer": {
+    "name": "platform-consumer-login"
+  },
+  "provider": {
+    "name": "platform-provider-login"
+  },
+  "interactions": [
+    {
+      "description": "logout and login fails with wrong password",
+      "providerState": "",
+      "request": {
+        "method": "POST",
+        "path": "/api/platform/admin_login",
+        "headers": {
+          "accept": "application/json, text/plain, */*",
+          "content-length": "38",
+          "content-type": "application/x-www-form-urlencoded"
+        },
+        "body": "username=admin&password=wrong-password",
+        "query": ""
+      },
+      "response": {
+        "status": 401,
+        "headers": {
+          "content-type": "application/json"
+        },
+        "body": {
+          "code": 401,
+          "message": "Invalid credentials."
+        },
+        "matchingRules": {
+          "$.body.code": {
+            "match": "type"
+          },
+          "$.body.message": {
+            "match": "type"
+          }
+        }
+      }
+    },
+    {
+      "description": "login",
+      "providerState": "",
+      "request": {
+        "method": "POST",
+        "path": "/api/platform/admin_login",
+        "headers": {
+          "accept": "application/json, text/plain, */*",
+          "content-length": "38",
+          "content-type": "application/x-www-form-urlencoded"
+        },
+        "body": "username=irontec&password=ee4aeVie7AiH",
```

Comment:
Let's use this opportunity to remove this credentials from our tests

Reply:
and what credentials should I use? You mean to modify those credentials, this kind of test are autogenerated between API response and cypress tests, so if we want to remove it, we should remove the cypress fixture as well and we won't be able to perform the login action.